### DISCUSSION TITLE
PrtSc secondary key always disabled

### DISCUSSION THREAD

**Discussion by sarodriguezco** [BartoszCichecki/LenovoLegionToolkit#1079] [Ideas]
Hi, thank you for your incredible work. 

I was wondering if it would be possible to have the PrtSc secondary key always disabled. In Windows, the PrtSc key can be automatically set up to take a screen capture. However, when the Fn lock is active, the Snipping Tool is opened instead. It's not a significant issue, but as someone who frequently takes screenshots, it would be great to be able to have the Fn lock and also maintain the default behavior of the PrtSc key.

Thanks again :)

**Comment by BartoszCichecki** [comment]
On most devices this key is hardcoded in firmware to send Win+Shift+S combo with Fn Lock so it isn't possible to change.

**Reply by sarodriguezco** [reply]
Thanks for your quick answer!
So to be clear, it is impossible to stop snipping tool from launching when pressing PrtSc and instead open directly the "take screenshot" feature (same as win+shift+s combo)?
